odoo/service/server.py

```diff
@@ -427,7 +427,7 @@ def cron_thread(self, number):
         # same time. This is known as the thundering herd effect.
 
         from odoo.addons.base.models.ir_cron import ir_cron
-        conn = odoo.sql_db.db_connect('postgres')
+        conn = odoo.sql_db.db_connect(odoo.tools.config['db_maintenance'])
```

Comment:
Didn't you forgot to use the correct database for the corresponding notify in ir_cron.py ?